GripOpGras2.Specs/Features/RationAlgorithmV1.feature

```diff
@@ -0,0 +1,102 @@
+| silo 1-2019 | 161.56            | 861.56        | maissilo 2018 | 81.82             | 959.39        | PERSPULP 24%   | 95.24                  | 1037.62            | GEPLETTE TARWE | 117.14                 | 1219.29            | TARWE MEEL     | 121.11                 | 1240               | 59        | 123.9        | 903.81    | 180           | 25.4           | 725.7                  | 806.58                 | 259.6                       | 309.41                      | 151925             | 162053.33          | 998870         | 1098757        |
+
+Scenario: Create a ration for a herd, when having two roughage products and one supplementary product
+	Given I have the roughage product <roughage1> that contains <protein-roughage1> g protein, and <vem-roughage1> VEM
+	And I have the roughage product <roughage2> that contains <protein-roughage2> g protein, and <vem-roughage2> VEM
+	And I have the supplementary product <supplementary1> that contains <protein-supplementary1> g protein, and <vem-supplementary1> VEM
+	And I have a herd with <herd-size> cows in it, which have taken in <grass-intake> kg dm grass
+	And each kg dm grass contains <vem-grass> VEM and <protein-grass> g protein
+	And my herd has produced <lmilk-produced> liters of milk
+	When I let Grip op Gras 2 create a ration
+    Then the ration should contain between <dm-roughage-ration-min> and <dm-roughage-ration-max> kg dm of roughage products
+    And the ration should contain between <dm-supplementary-ration-min> and <dm-supplementary-ration-max> kg dm of supplementary products
+    And the ration should contain between <protein-ration-min> and <protein-ration-max> g protein
+    And the ration should contain between <vem-ration-min> and <vem-ration-max> VEM
+	And the ration must contain <grass-intake> kg of grass
+Examples:
+| roughage1 | protein-roughage1 | vem-roughage1 | roughage2 | protein-roughage2 | vem-roughage2 | supplementary1 | protein-supplementary1 | vem-supplementary1 | herd-size | grass-intake | vem-grass | protein-grass | lmilk-produced | dm-roughage-ration-min | dm-roughage-ration-max | dm-supplementary-ration-min | dm-supplementary-ration-max | protein-ration-min | protein-ration-max | vem-ration-min | vem-ration-max |
+
+
+Scenario: Create a ration for a herd, without having any feed products
+	Given I have a herd with <herd-size> cows in it, which have taken in <grass-intake> kg dm grass
+	And each kg dm grass contains <vem-grass> VEM and <protein-grass> g protein
+	When I let Grip op Gras 2 create a ration
+	Then the ration must contain <grass-intake> kg of grass
+Examples: 
+| herd-size | grass-intake | vem-grass | protein-grass |
+| 200       | 100          | 1000      | 200           |
+| 100       | 0            | 10        | 200           |
+| 100       | 100          | 0         | 0             |
```

Comment:
Ik weet niet meer of dit besproken was of niet, maar in het geval van niet mijn comment:
zou je niet een melding/error moeten krijgen ipv een rantsoen?